### DISCUSSION TITLE
AI picture of recipe

### DISCUSSION THREAD

**Discussion by ThrawnJL** [mealie-recipes/mealie#4689] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Sometimes I have recipes with no pictures. Can’t find any that fits on google

### Please provide a concise description of the feature that would resolve your issue.

I have been using ChatGPT to make a picure, but copy pasting the text is annoying.
Could be nice if mealie itself could use the ai api to make pictures based on the recipes.

### Please consider and list out some caveats or tradeoffs made in your design decision

Probably going to be expensive to do from the open ai api.
Not sure if others would find this useful 

only an idea since I find the entire ai part of mealie very clever and useful 

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by ChipSkylark37** [comment]
I've been struggling with this as well. Sometimes I don't have a _good_ picture yet since I haven't made the recipe, and I tend to try and use pictures of the actual dish I've made (both as the recipe picture, and adding to the timeline).

Could you not just generate with one of the AI engines and use 'Copy Link' to get the image? 

When I really just want an image, I use the 'Copy Link' from a Google Image and then add it inside Mealie - No need for the text conversion or maybe I dont understand what you mean by that?

**Reply by ThrawnJL** [reply]
It might be b

> I've been struggling with this as well. Sometimes I don't have a _good_ picture yet since I haven't made the recipe, and I tend to try and use pictures of the actual dish I've made (both as the recipe picture, and adding to the timeline).
> 
> Could you not just generate with one of the AI engines and use 'Copy Link' to get the image?
> 
> When I really just want an image, I use the 'Copy Link' from a Google Image and then add it inside Mealie - No need for the text conversion or maybe I dont understand what you mean by that?

It might be me who use chatgpt wrong :) but I try and show it the recipe and then ask it to make a picture from that. I found a way to do it though ….coping text from mealies print view :)

**Reply by ChipSkylark37** [reply]
Oh, I understand what you are saying now. 

I do agree a 'generate image based on recipe & ingredients' would be neat to have.
